content/1-introduction-2-review.md

```diff
@@ -0,0 +1,636 @@
+The approach also has deep links with the analysis of *Niche Construction* [e.g. @odling-smee_niche_2003], which offers promising steps towards a reconciliation with non-informational views of culture stressing the importance of the development process in evolution.
+I return to this subject in more detail in @sec:intro-review-developments.
+
+\add{mesoudi-cultural-2011?}
+
+
+#### Parisian {#sec:intro-review-parisian}
+
+In the mid-nineties Dan Sperber formalised a second influential approach to the question of the evolution of culture:
+in a series of innovative articles gathered in @sperber_explaining_1996, the author puts forward a research programme called *Epidemiology of Representations* (better known today as *Cultural Attraction Theory*, hereafter noted CAT), and seeks to provide the cognitive and social sciences with a common framework with which to address interdisciplinary questions.
+One of the guiding questions of Sperber's work is the following: how can we explain both the diversity of culture across regions, and its relative stability through time, knowing that all human beings are more or less made of the same ingredients?
+In developing an answer, Sperber commits himself to presenting a coherent ontology where the status of each object he refers to is well defined, while at the same time connecting with the many ontologies he identifies in anthropology.
+
+The framework he suggests then starts from an ontology made of "mental representations", which correspond to those defined and studied by classical cognitive science, and "public representations", which are the expressions of mental representations in diverse cultural artefacts such as pieces of text, utterances, pictures, myths, built structures, etc..
+New mental representations are constantly formed in people's minds whenever they perceive or interpret public representations.
+For instance, say I am thinking of a tune (mental representation), and I whistle it (public representation);
+someone else hears it, and forms their own mental representation.
+Most of the time, the new representation in that person's head is different from my original representation.
+This last point is a defining feature of the theory Sperber proposes, in contrast to not only memetics, but also to SCE as outlined above [@sperber_explaining_1996, p. 25-26, 31].
+
+On this basis, Sperber proposes to model human societies as large dynamical systems of people continuously interpreting public representations into mental representations, and producing new public representations through their situated actions (in which mental representations play a role).
+To explain culture then, in this framework, is to analyse the processes by which representations circulate through a society, with different levels of change along the way.
```

Comment:
En effet. Donc en continuation du commentaire [ci-dessus](https://github.com/wehlutyk/thesis/pull/3#discussion_r124493859), je dirais que le point trois correspond à : 

> Ideas can be transmitted, and, by being transmitted from one person to another, they may even propagate. Some ideas - religious beliefs, cooking recipes, or scientific hypotheses, for instance - propagate so effectively that, in different versions, they may end up durably invading whole populations. Culture is made up, first and foremost, of such contagious ideas. It is made up also of all the productions (writings, artworks, tools, etc.) the presence of which in the shared environment of a human group permits the propagation of ideas.
(Sperber 1996, p. 1)

Alors la question de l'explication devient :

> 'To explain' may be taken in two senses. In the first one, to explain a cultural representation - for instance, a sacred text - is to make it intelligible - that is, to interpret it. [...] In the second sense, to explain a cultural representation is to show how it results from relatively general mechanisms at work in a given specific situation. In this second sense, the only one to be considered in this section, the explanation of cultural representations has an essential theoretical aspect: the identification of the general mechanisms at work
(Sperber 1996, p. 41)

> Representations are more or less widely and durably distributed, and hence more or less cultural. Under such conditions, to explain the cultural character of some representations amounts to answering the following question: Why are these representations more successful than others in a given human population? And in order to answer this question, the distribution of all representations, must be considered.
(Sperber 1996, p. 49)

Puis j'introduis les attracteurs comme outil explicatif.

Puis, comme il y a une diversité de domaines culturels (e.g. les techniques de construction d'un pont, les histoires telles que Le Petit Chaperon Rouge, etc.), l'explication d'un attracteur fera appel à différents modes de transmission et différents facteurs cognitifs et écologiques dans chaque domaine, bien que basés sur le même ensemble de mécanismes, modules, et susceptibilités cognitifs. (Indeed, merci d'avoir noté ça aussi.)

Pour these/those: tu y mets plus que simplement this/that?

TODO (modulo approval)
- [ ] check uses of these/those
- [ ] finish the rework in three steps from above
- [ ] then introduce what "explaining is"
- [ ] then branch into attractors and their diversity